flutter/lib/src/default_integrations.dart

```diff
@@ -370,15 +370,27 @@ class LoadReleaseIntegration extends Integration<SentryFlutterOptions> {
         if (name.isEmpty) {
```

Comment:
```suggestion
        if (name.isNotEmpty) {
```

Reply:
I don't think so. It would mean that we always use the app name if the package name is not empty.

Reply:
`var name = packageInfo.packageName;` This was the culprit.